### PR TITLE
site: add ga send event

### PR DESCRIPTION
**Pull request by chenshuai2144**
增加一些 统计信息
#### Checklist

- [x] `npm test` passes
- [x] tests are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

- any feature?
- close https://github.com/umijs/umi/ISSUE_URL

---
This change is [Reviewable](https://reviewable.io/reviews/umijs/umi/3395)

### CODE REVIEW

File: packages/umi-build-dev/src/plugins/commands/block/ui/uiUtil.ts [hunk 1]
```diff
@@ -154,3 +155,33 @@ export const getNoExitPath = async ({
...
 };
+
+/**
+ * google 发送统计信息
+ * @param param0
+ */
+export const sendGaEvent = ({ label, action }: { label: string; action: string }) => {
+  const ga = window && window.ga;
+  if (ga) {
+    ga('send', 'event', {
```
Comment (ycjcl868) [inline packages/umi-build-dev/src/plugins/commands/block/ui/uiUtil.ts]: 用 `gtag`

Comment (chenshuai2144) [reply packages/umi-build-dev/src/plugins/commands/block/ui/uiUtil.ts]: 好麻烦呀，你封装一下吧。
新的 api 改动还不小

Comment (ycjcl868) [reply packages/umi-build-dev/src/plugins/commands/block/ui/uiUtil.ts]: ga 和 gtag 都传给你了，先使用 ga 也可以。不建议在 ga gtag 做封装，抹平 api 差异

File: packages/umi-build-dev/src/plugins/commands/block/ui/ui/model/index.ts [hunk 2]
```diff
@@ -33,6 +34,12 @@ export default {
       if (blockData[resourceId] && !reload) {
         return blockData[resourceId];
       }
+
+      sendGaEvent({
```
Comment (ycjcl868) [inline packages/umi-build-dev/src/plugins/commands/block/ui/ui/model/index.ts]: dva effects 处理 ga 不是太好

**Pull request closed without merging**